### PR TITLE
Making the process of spurning up a basic Weblogic Domain (without operator) direct

### LINKED ISSUE #770: No metrics on prometheus
@mriccell So I'm at the step where I've deployed and started the weblogic servers and the managed servers on the cluster and have also run the command to create the prometheus pod. I've got a prometheus pod running 

```
[opc@kube-master-01 ~]$ kubectl get pods
NAME                         READY     STATUS    RESTARTS   AGE
ms-0                         1/1       Running   0          1d
ms-1                         1/1       Running   0          1d
prometheus-dfcb484b5-q2brd   1/1       Running   0          1d
wls-admin-server-0           2/2       Running   0          1d
```

but dropdown menu is empty:

image

So I went in and looked at the prometheus logs ot see if I could find anything there :

```
$ kubectl logs prometheus-dfcb484b5-q2brd
time="2018-02-22T16:34:02Z" level=error msg="github.com/prometheus/prometheus/discovery/kubernetes/kubernetes.go:207: Failed to list *v1.Pod: pods is forbidden: User "system:serviceaccount:default:default" cannot list pods at the cluster scope" component="kube_client_runtime" source="kubernetes.go:75"
time="2018-02-22T16:34:03Z" level=error msg="github.com/prometheus/prometheus/discovery/kubernetes/kubernetes.go:207: Failed to list *v1.Pod: pods is forbidden: User "system:serviceaccount:default:default" cannot list pods at the cluster scope" component="kube_client_runtime" source="kubernetes.go:75"
time="2018-02-22T16:34:04Z" level=error msg="github.com/prometheus/prometheus/discovery/kubernetes/kubernetes.go:207: Failed to list *v1.Pod: pods is forbidden: User "system:serviceaccount:default:default" cannot list p^C
```
So i went ahead and checked the prometheus configuration to find that it uses a prometheus.yml file (~~and I couldn't locate it anywhere in the git repo, assuming that it should have some wls specific config in it and hence it should be in the repo~~) :

```
$ kubectl exec prometheus-dfcb484b5-q2brd --stdin --tty -c prometheus /bin/sh

/prometheus # cat /etc/prometheus/prometheus.yml
global:
  scrape_interval:     5s
  external_labels:
    monitor: 'my-monitor'
scrape_configs:
- job_name: 'kubernetes-pods'
  kubernetes_sd_configs:
  - role: pod
  relabel_configs:
  - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_scrape]
    action: keep
    regex: true
  - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_path]
    action: replace
    target_label: __metrics_path__
    regex: (.+)
  - source_labels: [__address__, __meta_kubernetes_pod_annotation_prometheus_io_port]
    action: replace
    regex: ([^:]+)(?::\d+)?;(\d+)
    replacement: $1:$2
    target_label: __address__
  - action: labelmap
    regex: __meta_kubernetes_pod_label_(.+)
  - source_labels: [__meta_kubernetes_pod_name]
    action: replace
    target_label: pod_name
  - regex: '(controller_revision_hash|job)'
    action: labeldrop
  - source_labels: [name]
    regex: '.*/(.*)$'
    replacement: $1
    target_label: webapp
  basic_auth:
   username: weblogic
   password: weblogic1/prometheus #
```

and the prometheus app (navigating to Status--> Configuration) shows the config as : 
```
global:
  scrape_interval: 5s
  scrape_timeout: 5s
  evaluation_interval: 1m
  external_labels:
    monitor: my-monitor
scrape_configs:
- job_name: kubernetes-pods
  scrape_interval: 5s
  scrape_timeout: 5s
  metrics_path: /metrics
  scheme: http
  kubernetes_sd_configs:
  - api_server: null
    role: pod
    namespaces:
      names: []
  basic_auth:
    username: weblogic
    password: <secret>
  relabel_configs:
  - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_scrape]
    separator: ;
    regex: "true"
    replacement: $1
    action: keep
  - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_path]
    separator: ;
    regex: (.+)
    target_label: __metrics_path__
    replacement: $1
    action: replace
  - source_labels: [__address__, __meta_kubernetes_pod_annotation_prometheus_io_port]
    separator: ;
    regex: ([^:]+)(?::\d+)?;(\d+)
    target_label: __address__
    replacement: $1:$2
    action: replace
  - source_labels: []
    separator: ;
    regex: __meta_kubernetes_pod_label_(.+)
    replacement: $1
    action: labelmap
  - source_labels: [__meta_kubernetes_pod_name]
    separator: ;
    regex: (.*)
    target_label: pod_name
    replacement: $1
    action: replace
  - source_labels: []
    separator: ;
    regex: (controller_revision_hash|job)
    replacement: $1
    action: labeldrop
  - source_labels: [name]
    separator: ;
    regex: .*/(.*)$
    target_label: webapp
    replacement: $1
    action: replace
```

Not sure why prometheus is not showing the metrics. Any idea what I could be doing wrong?

### PR DESCRIPTION
**Pull request by karan-kapoor90**
This PR has changes to pull pre-built weblogic images from my docker repo as well as a few changes to the prometheus setup yml file to resolve #770

### CODE REVIEW

File: OracleWebLogic/samples/wls-K8s/k8s/wls-admin-webhook.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ spec:
     spec:
       containers:
       - name: webhook
-        image: oow-demo-webhook
+        image: karankapoor/oow-demo-webhook:latest
```
Comment (Djelibeybi) [inline OracleWebLogic/samples/wls-K8s/k8s/wls-admin-webhook.yml]: Can you change `karankapoor` to `oracle`, please?

File: OracleWebLogic/samples/wls-K8s/k8s/wls-admin-webhook.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ spec:
         - name: MS_STATEFULSET_NAME
           value: "ms"
       - name: wls-admin-server
-        image: wls-12213-oow-demo-domain
+        image: karankapoor/wls-12213-oow-demo-domain:latest
```
Comment (Djelibeybi) [inline OracleWebLogic/samples/wls-K8s/k8s/wls-admin-webhook.yml]: Can you change `karankapoor` to `oracle`, please?

File: OracleWebLogic/samples/wls-K8s/k8s/wls-stateful.yml [hunk 3]
```diff
@@ -17,7 +17,7 @@ spec:
       subdomain: wls-subdomain
       containers:
       - name: ms
-        image: wls-12213-oow-demo-domain
+        image: karankapoor/wls-12213-oow-demo-domain:latest
```
Comment (Djelibeybi) [inline OracleWebLogic/samples/wls-K8s/k8s/wls-stateful.yml]: Can you change `karankapoor` to `oracle`, please?

**Comment by mriccell** [comment]
@karan-kapoor90 could you please sign OCA so that we can merge your PR?

**Comment by Djelibeybi** [comment]
Closing due to lack of response.

**Pull request closed without merging**